### ISSUE TITLE
remote list

### ISSUE THREAD

**Issue by abidm8** [akshaynikhare/FireVisionIPTV#7]
m3u list should not be  built in as free lists do regularly change, therefor having a remote link would be more simpler to update.

**Comment by akshaynikhare** [comment]
@abidm8  You are absolutely correct , I need to do this , I have just not find the time , you are well come to do this if you like to. Contribute

**Comment by abidm8** [comment]
Thanks for your reply and offer, unfortunately Akshay I am no programmer,
only know the basics,
I kind of have my m3u list and was looking for something to embedded into,
yours seems great apart from that point I mentioned, I however came across
another project, very basic but not as good as yours, BUT , it has an m3u
link parser built-in.

I could find the link for that project and share it with you if you would
like to have a look.

Regards

Abid

**Comment by akshaynikhare** [comment]
@abidm8 yes, please share the link

**Comment by abidm8** [comment]
It is here

https://github.com/cgang/myiptv
